site/app/entities/course/CourseMaterial.php

```diff
@@ -38,7 +38,7 @@ class CourseMaterial {
 
     #[ORM\Column(type: Types::BOOLEAN)]
     protected bool $hidden_from_students;
-
+    protected bool $deleted;
     #[ORM\Column(type: Types::FLOAT)]
```

Comment:
This needs some additional whitespace around it, as well as a Doctrine attribute.